sudo make install fails

### Steps to reproduce

```bash
git clone https://github.com/warewulf/warewulf.git && \
    cd warewulf && \
    git checkout -b v4.6 v4.6.0rc1 && \
    make all && \
    sudo make install
```

### Error message

```
make: *** No rule to make target 'homemtimwarewulfvendorgithub.comspf13pflagbool.go', needed by 'wwctl'.  Stop.
```


### Information on your system

stock Rocky 9.5 system
warewulf v4.6.0rc1 build from git

```console
[mtim@head warewulf]$ wwctl version
wwctl version:   4.6.0rc1-1
rpc version: apiPrefix:"rc1" apiVersion:"1" warewulfVersion:"4.6.0rc1-1"
[mtim@head warewulf]$ cat /etc/os-release 
NAME="Rocky Linux"
VERSION="9.5 (Blue Onyx)"
ID="rocky"
ID_LIKE="rhel centos fedora"
VERSION_ID="9.5"
PLATFORM_ID="platform:el9"
PRETTY_NAME="Rocky Linux 9.5 (Blue Onyx)"
ANSI_COLOR="0;32"
LOGO="fedora-logo-icon"
CPE_NAME="cpe:/o:rocky:rocky:9::baseos"
HOME_URL="https://rockylinux.org/"
VENDOR_NAME="RESF"
VENDOR_URL="https://resf.org/"
BUG_REPORT_URL="https://bugs.rockylinux.org/"
SUPPORT_END="2032-05-31"
ROCKY_SUPPORT_PRODUCT="Rocky-Linux-9"
ROCKY_SUPPORT_PRODUCT_VERSION="9.5"
REDHAT_SUPPORT_PRODUCT="Rocky Linux"
REDHAT_SUPPORT_PRODUCT_VERSION="9.5"

```

### General information

- [x] I have run `wwctl version` and reported the contents of `/etc/os-release`
- [x] I have searched the issues of this repo and believe this is not a duplicate
- [x] I have captured and reported relevant error messages and logs

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -9,6 +9,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 ### Fixed
 
 - Fix default nodes.conf to use the new kernel command line list format. #1670
+- Fix `make install` when `sudo` does not set `$PWD`. #1660
 
 ## v4.6.0rc1, 2025-01-29
 

--- a/Variables.mk
+++ b/Variables.mk
@@ -75,7 +75,7 @@ CONFIG := $(shell pwd)
 IPXESOURCE ?= $(PREFIX)/share/ipxe
 
 # helper functions
-godeps=$(shell 2>/dev/null go list -mod vendor -deps -f '{{if not .Standard}}{{ $$dep := . }}{{range .GoFiles}}{{$$dep.Dir}}/{{.}} {{end}}{{end}}' $(1) | sed "s%${PWD}/%%g")
+godeps=$(shell 2>/dev/null go list -mod vendor -deps -f '{{if not .Standard}}{{ $$dep := . }}{{range .GoFiles}}{{$$dep.Dir}}/{{.}} {{end}}{{end}}' $(1) | sed "s%$(shell pwd)/%%g")
 
 # use GOPROXY for older git clients and speed up downloads
 GOPROXY ?= https://proxy.golang.org

